tests/drivers/spi/spi_loopback/src/spi.c

```diff
@@ -428,6 +429,79 @@ static int spi_rx_every_4(struct spi_dt_spec *spec)
 	return 0;
 }
 
+static int spi_rx_bigger_than_tx(struct spi_dt_spec *spec)
+{
+	const uint32_t tx_buf_size = 8;
+
+	_Static_assert(tx_buf_size < BUF_SIZE);
```

Comment:
```suggestion
	BUILD_ASSERT(tx_buf_size < BUF_SIZE, "Transmit buffer is expected to be smaller than the receive buffer");
```

Reply:
Didn't see the "commit suggestion" button so I applied the change manually.